CODEOWNERS

```diff
@@ -206,8 +206,10 @@
 /drivers/i2s/i2s_ll_stm32*                @avisconti
 /drivers/i2c/i2c_common.c                 @sjg20
 /drivers/i2c/i2c_emul.c                   @sjg20
+/drivers/i2c/i2c_ite_it8xxx2.c            @ite
```

Comment:
<at>ite is not a github handle.  Who is the codeowner?

Reply:
+1